pylops/signalprocessing/udct.py

```diff
@@ -0,0 +1,63 @@
+__all__ = ["UDCT"]
+
+import numpy as np
+
+from pylops import LinearOperator
+from pylops.utils import deps
+from pylops.utils.decorators import reshaped
+from pylops.utils.typing import NDArray
+from ucurv import udct, ucurvfwd, ucurvinv, bands2vec, vec2bands
+
+ucurv_message = deps.ucurv_import("the ucurv module")
+
+
+class UDCT(LinearOperator):
+    r"""Uniform Discrete Curvelet Transform
+
+    Perform the multidimensional discrete curvelet transforms
+
+    The UDCT operator is a wraparound of the ucurvfwd and ucurvinv
+    calls in the UCURV package. Refer to
+    https://ucurv.readthedocs.io for a detailed description of the
+    input parameters.
+
+    Parameters
```

Comment:
You have only 3 parameters here, but they should be those of the init method, which are 5 in your case